test/integration/targets/luks_device/tasks/tests/options.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Create with keysize
+  luks_device:
+    device: "{{ cryptfile_device }}"
+    state: present
+    keyfile: "{{ role_path }}/files/keyfile1"
+    keysize: 256
+  become: yes
+  register: create_with_keysize
+- name: Create with keysize (idempotent)
+  luks_device:
+    device: "{{ cryptfile_device }}"
+    state: present
+    keyfile: "{{ role_path }}/files/keyfile1"
+    keysize: 256
+  become: yes
+  register: create_idem_with_keysize
```

Comment:
Please add a third test where `keysize` has another value. This should also result in no change. (If we ever change this, we'll be reminded that the current behavior was expected and not a bug.)